jax/_src/numpy/reductions.py

```diff
@@ -422,9 +422,54 @@ def _reduce_all(a: ArrayLike, axis: Axis = None, out: None = None,
   return _reduction(a, "all", np.all, lax.bitwise_and, True, preproc=_cast_to_bool,
                     axis=axis, out=out, keepdims=keepdims, where_=where)
 
-@implements(np.all, skip_params=['out'])
+
 def all(a: ArrayLike, axis: Axis = None, out: None = None,
         keepdims: bool = False, *, where: ArrayLike | None = None) -> Array:
+  r"""Test whether all array elements along a given axis evaluate to True.
+
+  JAX implementation of :func:`numpy.all`.
+
+  Args:
+    a: Input array.
+    axis: int or array, default=None. Axis along which the test to be performed
+      to evaluate for True values. If None, tests along all the axes.
```

Comment:
Fix grammar in the sentence starting with "Axis"

Reply:
Fixed and squashed into single commit.